src/tiledb/cloud/bioimg/helpers.py

```diff
@@ -41,10 +41,18 @@ def is_folder(path: str) -> bool:
     return path.endswith("/")
 
 
-def validate_io_paths(source: Sequence[str], output: Sequence[str]) -> None:
+def validate_io_paths(
+    source: Sequence[str], output: Sequence[str], register: bool
```

Comment:
When there’s a boolean parameter, I find it almost always better to make it keyword-only, because in general there’s no way at the callsite to tell what `do_something(x, y, True)` means vs. `do_something(x, y, False)`. `for_registration` might be a better name for this parameter as well.

Reply:
Sure I missed this one. I thought that since validate is an internal usage function the caller that passes register as value would make it self explanatory but I will make it a named arg.